### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by leonardoeng13**
For this milestone of your portfolio website, you will implement a client-side validation of the data in the contact form before submission.

### Warning
- To test the mobile version, is necessary to start Chrome Dev Tools and select iPhone than a responsive option.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -21,6 +17,22 @@ const menuItem1 = document.querySelector('#menu-item1');
...
+const mobileSubmit = document.querySelector('#form-submit-mobile');
+const name_mobile = document.querySelector('#name-mobile');
+const email_mobile = document.querySelector('#email-mobile');
+const msg_mobile = document.querySelector('#msg-mobile');
+const errormsgmobile = document.querySelector('#errormsgmobile');
+var d_formInfo = {name:'', email:'', msg:''};
+var m_formInfo = {name:'', email:'', msg:''};
+var desktop_info = "";
+var mobile_info = ""; */
+// ################## END FORM VALIDATION ########################
```
Comment (Davidosky007) [inline js/script.js]: * Kindly remove uncommented code because it makes your codebase bulky and unprofessional

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @leonardoeng13 @herokudev 👋, 

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!
cr2 gif_almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: js/script.js (hunk 1 above)
Comment (leonardoeng13) [reply js/script.js]: Unnecessary comments removed.

**Review by oxenprogrammer — APPROVED** [review]
Hi **Leonardo**, 

Thanks for making the **_Requested Changes_**. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

__OPTIONAL__
- [ ] It is possible to improve on the **commit messages**. Good commit messages allows for easy collaboration and tracking of file Change-log. _Please checkout [this article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) on how and why to write professional commit messages._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**